sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Azure.ResourceManager.AppConfiguration.csproj

```diff
@@ -7,7 +7,8 @@
       This is a beta preview vesion. This version uses a next-generation code generator that introduces important breaking changes, but also new features (such as intuitive authentication, custom HTTP pipeline, distributed tracing and much more).
     </Description>
     <PackageTags>azure;management;appconfiguration</PackageTags>
-    <NoWarn>$(NoWarn);AZC0001;AZC0008;</NoWarn>
+    <NoWarn>$(NoWarn);AZC0008;</NoWarn>
```

Comment:
What is the plan for AZC0008?

Reply:
When that package gets updated to the new version that will go away since it will use the central ArmClient and ArmClientOptions